src/test/java/uk/gov/companieshouse/api/accounts/service/impl/IntangibleAssetsServiceTest.java

```diff
@@ -0,0 +1,331 @@
+package uk.gov.companieshouse.api.accounts.service.impl;
+
+import com.mongodb.MongoException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.verification.VerificationMode;
+import org.springframework.dao.DuplicateKeyException;
+import uk.gov.companieshouse.api.accounts.ResourceName;
+import uk.gov.companieshouse.api.accounts.exception.DataException;
+import uk.gov.companieshouse.api.accounts.links.BasicLinkType;
+import uk.gov.companieshouse.api.accounts.links.SmallFullLinkType;
+import uk.gov.companieshouse.api.accounts.model.entity.notes.intangible.IntangibleAssetsEntity;
+import uk.gov.companieshouse.api.accounts.model.rest.notes.intangible.IntangibleAssets;
+import uk.gov.companieshouse.api.accounts.repository.IntangibleAssetsRepository;
+import uk.gov.companieshouse.api.accounts.service.response.ResponseObject;
+import uk.gov.companieshouse.api.accounts.service.response.ResponseStatus;
+import uk.gov.companieshouse.api.accounts.transformer.IntangibleAssetsTransformer;
+import uk.gov.companieshouse.api.accounts.utility.impl.KeyIdGenerator;
+import uk.gov.companieshouse.api.model.transaction.Transaction;
+import uk.gov.companieshouse.api.model.transaction.TransactionLinks;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.Map;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Please can we use explicit imports?

Reply:
This was because we were having major issues with the junit version. We had to change it to this as it was automatically dragging in junit 4 which was crashing our tests.